code/_onclick/hud/movable_screen_objects.dm

```diff
@@ -20,6 +21,8 @@
 
 
 /obj/screen/movable/MouseDrop(over_object, src_location, over_location, src_control, over_control, params)
+	if(locked) //no! I am locked! begone!
+		return
```

Comment:
I think cyberboss was trying to ask if this disabled the dragging or just made it snap back into position after you finished dragging it.